### PR TITLE
Fix user list

### LINKED ISSUE #15583: Users page layout issue
On master (660aa7ff1e7bcb61d5dcb103014da39b8a257899)

It looks like the login name column is missing:
login_username_missing

@rullzer @MorrisJobke @LukasReschke

### PR DESCRIPTION
**Pull request by MorrisJobke**
- the name cell was changed from td to th because of accessibility
  issues - this adjusts the CSS and JS to behave like before

Fixes #15583

cc @owncloud/designers @nickvergessen @PVince81

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @volker-e

File: settings/js/users/users.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ var UserList = {
 		$tr.data('displayname', user.displayname);
 		$tr.data('mailAddress', user.email);
 		$tr.data('restoreDisabled', user.isRestoreDisabled);
-		$tr.find('td.name').text(user.name);
+		$tr.find('th.name').text(user.name);
```
Comment (nickvergessen) [inline settings/js/users/users.js]: why not fix the template file and add a close for the color? `<th>` inside `<tbody>` doesn't sound like a good idea to me..

**Comment by karlitschek** [comment]
:+1:

File: settings/js/users/users.js (hunk 1 above)
Comment (nickvergessen) [inline settings/js/users/users.js]: Okay, looks like this is actually what you should do:
http://stackoverflow.com/questions/5509163/does-html5-ban-th-cells-from-tbody
https://html.spec.whatwg.org/multipage/tables.html#the-th-element

**Comment by nickvergessen** [comment]
:+1: works again, would be nice to have tests (if somehow possible)

File: settings/js/users/users.js (hunk 1 above)
Comment (MorrisJobke) [inline settings/js/users/users.js]: What do you mean with this?

Comment (jancborchardt) [inline settings/js/users/users.js]: It’s best to just use `.name`. Then we don’t have to adjust this call when the tag is changed in the future.

File: settings/js/users/users.js [hunk 2]
```diff
@@ -259,8 +259,8 @@ var UserList = {
 		rows.sort(function(a, b) {
 			// FIXME: inefficient way of getting the names,
 			// better use a data attribute
-			a = $(a).find('td.name').text();
-			b = $(b).find('td.name').text();
+			a = $(a).find('th.name').text();
+			b = $(b).find('th.name').text();
 			var firstSort = UserList.preSortSearchString(a, b);
```
Comment (jancborchardt) [inline settings/js/users/users.js]: Here as well, just use `.name`.

**Comment by MorrisJobke** [comment]
@jancborchardt done ;)

**Pull request merged**